react/features/dial-out/actions.js

```diff
@@ -0,0 +1,97 @@
+import { openDialog } from '../../features/base/dialog';
+
+import {
+    DIAL_OUT_CANCELED,
+    DIAL_OUT_CODES_UPDATED,
+    DIAL_OUT_SERVICE_FAILED,
+    PHONE_NUMBER_CHECKED
+} from './actionTypes';
+
+import { DialOutDialog } from './components';
+
+declare var $: Function;
+declare var config: Object;
+
+/**
+ * Dials the given number.
+ *
+ * @returns {Function}
+ */
+export function cancel() {
+    return {
+        type: DIAL_OUT_CANCELED
+    };
+}
+
+/**
+ * Dials the given number.
+ *
+ * @param {string} dialNumber - The number to dial.
+ * @returns {Function}
+ */
+export function dial(dialNumber) {
+    return (dispatch, getState) => {
+        const { conference } = getState()['features/base/conference'];
+
+        conference.dial(dialNumber);
```

Comment:
This is something we'll need to add in lib-jitsi-meet. Currently the "error handling" that we do is only through presence messages and we turn them into notifications.